### DISCUSSION TITLE
Industry Session

### DISCUSSION THREAD

**Discussion by Abhishek-1228** [github/india#276] [Students]
The industry sessions show incomplete even though completed? Does anyone knows why?

**Comment by Mik-27** [comment]
Same issue.

**Comment by Mik-27** [comment]
Refer #261

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
